huxley/api/permissions.py

```diff
@@ -106,6 +106,19 @@ def has_permission(self, request, view):
 
 
 class DelegateListPermission(permissions.BasePermission):
+
+    def has_permission(self, request, view):
+        if request.user.is_superuser or request.method == 'POST':
+            return True
+
+        committee_id = view.request.GET.get('committee_id', None)
+        user = request.user
+
+        return (user.is_authenticated() and user.is_chair() and
+                user.assignment.committee.id == int(committee_id))
+
+
+class DelegateListPermission(permissions.BasePermission):
```

Comment:
This class is already defined, surprised that it's not a runtime error. Did you mean to name it something different above?